tests/RobotFramework/tests/cumulocity/self-update/tedge_self_update.robot

```diff
@@ -142,8 +136,15 @@ Create Local Repository
     Execute Command    cd /opt/repository/local && dpkg-scanpackages -m . > Packages
     Execute Command    cmd=echo 'deb [trusted=yes] file:/opt/repository/local /' > /etc/apt/sources.list.d/tedge-local.list
 
-Install Packages
-    [Arguments]    ${packages_dir}=/setup/packages
-
-    Execute Command    apt-get update && apt-get install -y --no-install-recommends mosquitto
-    Execute Command    cd ${packages_dir} && dpkg -i tedge_*.deb && dpkg -i tedge*mapper*.deb && dpkg -i tedge*agent*.deb && dpkg -i tedge*watchdog*.deb && dpkg -i tedge*apt*plugin*.deb
+Install fixed thin-edge.io version
+    [Documentation]    APT Package pinning is used to install a fixed version as specifying a version via `apt install tedge-full=1.0.0`
+            ...        does not work as tedge-full is a virtual packages and the version does not get propogated to the package dependencies.
+            ...        In the future the "thin-edge.io/install.sh" script might handle this automatically, however until then we can use
+            ...        Package pinning in the test.
+    [Arguments]    ${version}
+    Transfer To Device    ${CURDIR}/tedge-fixed-version.conf    /etc/apt/preferences.d/tedge-fixed-version.conf
+    Execute Command    sed -i 's/Pin: version .*/Pin: version ${version}/g' /etc/apt/preferences.d/tedge-fixed-version.conf
```

Comment:
Do you plan to pin & install a version that is not hard-coded in `tests/RobotFramework/tests/cumulocity/self-update/tedge-fixed-version.conf` ?

Reply:
The `1.0.0` in the `tests/RobotFramework/tests/cumulocity/self-update/tedge-fixed-version.conf` file is just a placeholder, then the sed will update it as defined in the test.

In the future, this "pinning" will be done in the `install.sh` script (once we get around to it)

Reply:
Good thing you prompted me to check...the pinning file is being ignored due to a naming problem...however I'm just thinking whether we actually need to pin it to a specific version, or whether we can just use the "latest official" version...

Reply:
@didier-wenzek  I would propose the following:
* Install the latest official version (currently 1.0.1) (no package pinning required)

Are you ok with this?

Reply:
I pushed the proposed change as overall this will be less maintenance overhead as the latest official version will always be used (and pulling older packages will not affect this). 3f1d1ffd1670f0c560d5cd77dc7d39a8d5a52c04